ci-jobs/scripts/build-webdriveragent.js

```diff
@@ -13,6 +13,14 @@ async function buildWebDriverAgent (xcodeVersion) {
   xcodeVersion = xcodeVersion || await xcode.getVersion();
   log.info(`Building bundle for Xcode version '${xcodeVersion}'`);
 
+  // Clear WebDriverAgent from derived data
```

Comment:
why this is needed?

Reply:
how about using the standard `xcodebuild clean` operation instead?

Reply:
@mykola-mokhnach because if the user's machine had multiple builds of WebDriverAgent an old build may be pulled in. This guarantees that derivedData/WebDriverAgent is cleared and then when a new one is built, there's only one there.